core/vtk/ttkScalarFieldCriticalPoints/ttkScalarFieldCriticalPoints.cpp

```diff
@@ -78,6 +82,17 @@ int ttkScalarFieldCriticalPoints::RequestData(
   this->preconditionTriangulation(triangulation);
   this->setOutput(&criticalPoints_);
 
+#if TTK_ENABLE_MPI_TIME
```

Comment:
Please create a function `startMPItimer`, to declare and implement in `core/vtk/ttkAlgorithm/ttkUtils.h` and `core/vtk/ttkAlgorithm/ttkUtils.cpp`.

This entire snippet of code (from line 85 to 94, with the `#if` and `#endif` included) should be put altogether in that function.